src/Kernel/Variable.class.st

```diff
@@ -175,6 +194,11 @@ Variable >> isTemp [
 	^ false
 ]
 
+{ #category : #testing }
+Variable >> isTemporaryVariable [
```

Comment:
I think a shorter version will be better: #isTempVariable. 
I would also rename TemporaryVariable to TempVariable. 
Otherwise we should use longer names for subclasses to be consistent (OCCopyingTempVariable. OCVectorTempVariable )

Reply:
yes, possible... there are already #hasTemporaryVariableNamed: (and similar), but they where not used a lot

What is important is to not shorten "Variable" to "Var" as that is the difference between old and new reflective methods (e.g. instVarNamed: vs instanceVariableNamed:)